packages/react-data-grid/src/masks/InteractionMasks.js

```diff
@@ -139,11 +140,24 @@ class InteractionMasks extends React.Component {
     this.unsubscribeDragEnter();
   }
 
-  saveEditorPosition = () => {
+  getEditorPosition = () => {
     if (this.selectionMask) {
-      const { left, top } = this.selectionMask.getBoundingClientRect();
-      const { scrollLeft, scrollTop } = document.documentElement;
-      this.editorPosition = { left: left + scrollLeft, top: top + scrollTop };
+      const { editorPortalTarget } = this.props;
+      const { left: selectionMaskLeft, top: selectionMaskTop } = this.selectionMask.getBoundingClientRect();
+      if (editorPortalTarget === document.body) {
+        const { scrollLeft, scrollTop } = document.scrollingElement || document.documentElement;
+        return {
+          left: selectionMaskLeft + scrollLeft,
+          top: selectionMaskTop + scrollTop
+        };
+      }
+
+      const { left: portalTargetLeft, top: portalTargetTop } = editorPortalTarget.getBoundingClientRect();
```

Comment:
@MayhemYDG I am only checking `editorPortalTarget` for now as checking all the parent components were causing some issues and I do not expect react data grid to be used in multiple nested scrolling parents. If a need arises then we can revisit this logic.